gulpfile.js

```diff
@@ -5,21 +5,142 @@
 
 'use strict';
 
+const fs = require('fs');
+const path = require('path');
 const del = require('del');
 const gulp = require('gulp');
+const gulpUtil = require('gulp-util');
 const tslint = require('gulp-tslint');
 const vsce = require('vsce');
-//const omnisharpDownload = require('./out/omnisharpDownload');
+const debugUtil = require('./out/coreclr-debug/util.js');
+const debugInstall = require('./out/coreclr-debug/install.js');
+const fs_extra = require('fs-extra-promise');
+const omnisharpDownload = require('./out/omnisharpDownload');
+const child_process = require('child_process');
+
+const OmniSharpVersion = omnisharpDownload.OmniSharpVersion;
+
+/// used in offline packaging run so does not clean .vsix
+function clean() {
+    cleanDebugger();
+    return cleanOmnisharp();
+}
+
+gulp.task('clean', ['omnisharp:clean',  'debugger:clean', 'package:clean'], () => {
+
+});
+
+/// Omnisharp Tasks
+function installOmnisharp(omnisharpAssetName) {
+    const logFunction = (message) => { console.log(message); };
```

Comment:
This is arrow function and other callbacks use the `function(...) { ... }` syntax. Should we be consistent?